pre-commit/notebook_urls.py

```diff
@@ -204,4 +209,4 @@ def replace_line(line: str, to_ref: str, replacements: List[PatternReplacement])
         default=git_branch_name(),
     )
     args = parser.parse_args()
-    sys.exit(1 - main(args.files, args.from_ref, args.to_ref))
+    sys.exit(main(args.files, args.from_ref, args.to_ref))
```

Comment:
Why is this changed? It will cause the program to exit with code 1 when passing. It should be 0 on success.

Reply:
Generating an error where? It's supposed to be 0 on success

Reply:
OK, now it runs fine.